doc/classes/OS.xml

```diff
@@ -716,82 +728,121 @@
 	</members>
 	<constants>
 		<constant name="DAY_SUNDAY" value="0" enum="Weekday">
+			Sunday.
 		</constant>
 		<constant name="DAY_MONDAY" value="1" enum="Weekday">
+			Monday.
 		</constant>
 		<constant name="DAY_TUESDAY" value="2" enum="Weekday">
+			Tuesday.
 		</constant>
 		<constant name="DAY_WEDNESDAY" value="3" enum="Weekday">
+			Wednesday.
 		</constant>
 		<constant name="DAY_THURSDAY" value="4" enum="Weekday">
+			Thursday.
 		</constant>
 		<constant name="DAY_FRIDAY" value="5" enum="Weekday">
+			Friday.
 		</constant>
 		<constant name="DAY_SATURDAY" value="6" enum="Weekday">
+			Saturday.
 		</constant>
 		<constant name="MONTH_JANUARY" value="1" enum="Month">
+			January.
 		</constant>
 		<constant name="MONTH_FEBRUARY" value="2" enum="Month">
+			February.
 		</constant>
 		<constant name="MONTH_MARCH" value="3" enum="Month">
+			March.
 		</constant>
 		<constant name="MONTH_APRIL" value="4" enum="Month">
+			April.
 		</constant>
 		<constant name="MONTH_MAY" value="5" enum="Month">
+			May.
 		</constant>
 		<constant name="MONTH_JUNE" value="6" enum="Month">
+			June.
 		</constant>
 		<constant name="MONTH_JULY" value="7" enum="Month">
+			July.
 		</constant>
 		<constant name="MONTH_AUGUST" value="8" enum="Month">
+			August.
 		</constant>
 		<constant name="MONTH_SEPTEMBER" value="9" enum="Month">
+			September.
 		</constant>
 		<constant name="MONTH_OCTOBER" value="10" enum="Month">
+			October.
 		</constant>
 		<constant name="MONTH_NOVEMBER" value="11" enum="Month">
+			November.
 		</constant>
 		<constant name="MONTH_DECEMBER" value="12" enum="Month">
+			December.
 		</constant>
 		<constant name="SCREEN_ORIENTATION_LANDSCAPE" value="0" enum="ScreenOrientation">
+			Landscape screen orientation.
 		</constant>
 		<constant name="SCREEN_ORIENTATION_PORTRAIT" value="1" enum="ScreenOrientation">
+			Portrait screen orientation.
 		</constant>
 		<constant name="SCREEN_ORIENTATION_REVERSE_LANDSCAPE" value="2" enum="ScreenOrientation">
+			Reverse landscape screen orientation.
 		</constant>
 		<constant name="SCREEN_ORIENTATION_REVERSE_PORTRAIT" value="3" enum="ScreenOrientation">
+			Reverse portrait screen orientation.
 		</constant>
 		<constant name="SCREEN_ORIENTATION_SENSOR_LANDSCAPE" value="4" enum="ScreenOrientation">
+			Either landscape or reverse landscape screen orientation depending on the direction the screen is facing as determined by the hardware sensor.
```

Comment:
The "depending" clause feels overly wordy.

Uses landscape or reverse landscape based on the hardware sensor.